.pylintrc

```diff
@@ -233,7 +233,7 @@ single-line-if-stmt=no
 no-space-check=trailing-comma,dict-separator
 
 # Maximum number of lines in a module
-max-module-lines=1000
+max-module-lines=1100 # TODO: refactor py/kubeflow/testing/util.py. Set it back to 1000
```

Comment:
Can you file a GitHub issue to track this please? And then link the gitHub issue here.